_SETUP.md

```diff
@@ -21,8 +21,20 @@ yarn link
 yarn link webpack
 ```
 
-To run the entire test suite use:
+### To run the entire test suite use:
 
 ```bash
 yarn test
 ```
+
+### To run the typechecker use:
+
+```bash
+yarn type-lint
+```
+
+or incremental as you add JSDoc Annotations or make changes
+
+```bash
+yarn type-lint --watch
+```
```

Comment:
Also add notes about the other commands:

`yarn lint` `yarn code-lint` `yarn test:integration` `yarn test:integration --watch` `yarn test:unit` `yarn pretty`

Reply:
Hehe, I saw this coming. :joy: